### PR TITLE
Jellybean

### PR DESCRIPTION
**Pull request by mcdope**
Heya,

First: Thanks for your awesome work :)

I've added support for the LG P720 (Optimus 3D Max), maybe you wanna merge it... 
If there is a quality issue for you with the changes, I can fix it / change it. Just tell me...

Greets, Tobias

### CODE REVIEW

**Comment by szlkiev** [comment]
This pull request cannot be automatically merged.

please rebase

**Comment by mcdope** [comment]
Should work now. Please try again.

File: products/AndroidProducts.mk [hunk 1]
```diff
@@ -67,6 +67,9 @@ endif
 ifeq (pac_vigor,$(TARGET_PRODUCT))
     PRODUCT_MAKEFILES += $(LOCAL_DIR)/pac_vigor.mk
 endif
+ifeq (pac_p720,$(TARGET_PRODUCT))
+    PRODUCT_MAKEFILES += $(LOCAL_DIR)/pac_p720.mk
+endif
```
Comment (mcdope) [inline products/AndroidProducts.mk:72]: Manually merged conflict

File: vendorsetup.sh [hunk 2]
```diff
@@ -24,8 +24,11 @@ add_lunch_combo pac_gio_userdebug
 add_lunch_combo pac_grouper-userdebug
 # HTC Rezound
 add_lunch_combo pac_vigor-userdebug
+# LG Electronics Optimus 3D MAX (P720)
+add_lunch_combo pac_p720-userdebug
```
Comment (mcdope) [inline vendorsetup.sh:28]: Manually merged conflict

**Pull request merged**